packages/map-template/src/components/Map/Map.jsx

```diff
@@ -49,35 +36,26 @@ let _tileStyle;
  * @param {boolean} props.useMapProviderModule - If you want to use the Map Provider set on your solution in the MapsIndoors CMS, set this to true.
  * @returns
  */
-function Map({ onLocationClick, onVenueChangedOnMap, useMapProviderModule }) {
+function Map({ onLocationClick, onVenueChangedOnMap, useMapProviderModule, onMapPositionKnown }) {
```

Comment:
Add function docs for the `onMapPositionKnown` prop 😄

Reply:
🫡